collector/src/main/java/com/bitmovin/analytics/ads/Ad.kt

```diff
@@ -0,0 +1,35 @@
+package com.bitmovin.analytics.ads
+
+data class Ad(var isLinear: Boolean? = null,
```

Comment:
We should re-visit if really every single one of these can be nullable or not. Might make sense to have some non-nullables here that are always guaranteed to be set.. (Could also be a last step once we have the impl nailed down)

Reply:
Agree here, i can remove the nullable from those values directly convertable from the Ad interface of the player
we should keep nullable for every other fields which are cast depending or based on nullabel values 
What do you think?